### PR TITLE
TELCODOCS-1165 - Support multiple storage/device classes

### PR DESCRIPTION
**Pull request by aireilly**
Adds docs for multiple storage/device classes support for LVM Storage.

Version(s):
enterprise-4.13

Issue:
https://issues.redhat.com/browse/TELCODOCS-1165

Link to docs preview:
https://56366--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.html#lvms-creating-lvms-cluster_logical-volume-manager-storage

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

**Review by kcarmichael08 — COMMENTED** [review]
Can you please squash to one commit?

File: modules/lvms-creating-an-logical-volume-manager-cluster.adoc [hunk 1]
```diff
@@ -4,78 +4,203 @@
...
 
-{product-title} supports additional worker nodes for {sno} clusters on bare-metal user-provisioned infrastructure.
-{lvms} detects and uses the additional worker nodes when the new nodes show up.
-In case you need to set a node filter for the additional worker nodes, you can use the YAML view while creating the cluster.
+[NOTE]
+====
+When you create a Logical Volume Manager cluster, `StorageClass` and `LVMVolumeGroup` resources work together to provide dynamic provisioning of storage.
+`StorageClass` CRs define the properties of the storage that you can dynamically provision.
+`LVMVolumeGroup` is a specific type of persistent volume (PV) that is backed by an LVM Volume Group.
+`LVMVolumeGroup` CRs provide the backend storage for the persistent volumes that you create.
```
Comment (kcarmichael08) [inline modules/lvms-creating-an-logical-volume-manager-cluster.adoc:17]: IBM Style suggests "back-end" (usually hyphenated) - I found both hyphenated and non-hyphenated versions in the OCP docs repo - more hyphenated than not, so I'm not sure about this particular usage. 
```suggestion
`LVMVolumeGroup` CRs provide the back-end storage for the persistent volumes that you create.
```

**Review by kcarmichael08 — COMMENTED** [review]
You did a great job with the writing and the flow of info! I had some comment/questions but it looks really good.

**Review by skrthomas — COMMENTED** [review]
Merge review LGTM

**Pull request merged**